static/templates/stream_settings/stream_settings.hbs

```diff
@@ -3,17 +3,17 @@
     {{#with sub}}
     <div class="button-group">
         <div class="sub_unsub_button_wrapper inline-block">
-            <button class="button small rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" {{#if should_display_subscription_button}}title="{{t 'Toggle subscription'}} (S)" {{else}}disabled="disabled"{{/if}}>
+            <button class="button small rounded subscribe-button sub_unsub_button tippy-zulip-tooltip {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button" {{#if should_display_subscription_button}}data-tippy-content="{{t 'Toggle subscription'}} (S)" {{else}}disabled="disabled"{{/if}}>
```

Comment:
What happens if we add this class but don't set `data-tippy-tooltip`? Seems like that happens if `!should_display_subscription_button`.

Reply:
@timabbott when exactly is `should_display_subscription_button` set to false? Because i could recreate this..
But i tried it with removing `data-tippy-tooltip` completely while keeping `tippy-zulip-tooltip`, this works without any problems, because if data-tippy-tooltip is not set here, the button is disabled, so if the user hovers over it they just see the symbol for disabled buttons.

Reply:
I think the main situation is if you're an administrator looking at a stream you're not subscribed to in "all streams".
```
static/js/stream_settings_data.js:    sub.should_display_subscription_button = stream_data.can_toggle_subscription(sub);
```


In any case, it seemed more correct to just add a second `if` statement around setting the tooltip property too, so I did that and merged this.